Mail/Views/Thread/WebView/WebViewModel.swift

```diff
@@ -37,6 +37,7 @@ final class WebViewModel: NSObject, ObservableObject {
     let contentBlocker: ContentBlocker
 
     let style: String = MessageWebViewUtils.generateCSS(for: .message)
```

Comment:
This property is not necessary anymore as you declared it in the `WebViewModelOperations` actor